spec/models/term_spec.rb

```diff
@@ -105,34 +133,49 @@
   end
 
   describe 'when creating a term' do
-    it 'fails if pref_label missing' do
-      expect {
-        FactoryBot.create(:external_term, pref_label: nil)
-      }.to raise_error ActiveRecord::RecordInvalid
+    context 'with missing pref_label' do
+      let(:term) { FactoryBot.build(:external_term, pref_label: nil) }
+
+      it 'returns validation error' do
+        expect(term.save).to be false
+        expect(term.errors.full_messages).to include 'Pref label can\'t be blank'
+      end
     end
 
-    it 'fails if term_type missing' do
-      expect {
-        FactoryBot.create(:external_term, term_type: nil)
-      }.to raise_error ActiveRecord::RecordInvalid
+    context 'with missing term_type' do
+      let(:term) { FactoryBot.build(:external_term, term_type: nil) }
+
+      it 'returns validation error' do
+        expect(term.save).to be false
+        expect(term.errors.full_messages).to include('Term type can\'t be blank', 'Term type is not included in the list')
```

Comment:
```suggestion
        expect(term.errors.full_messages).to include('Term type can\'t be blank', 'Invalid term type: nil')
```

Reply:
Instead of `message: 'is not valid: %{value}'`, does this work?

`message: ->(object, data) { "is not valid: #{data[:value].nil? ? '(no value) : data[:value] }"}

(https://guides.rubyonrails.org/active_record_validations.html#message)

Reply:
I'm approving this PR in advance of any `message` change since everything else looks good.  It would be great if my above suggestion works, but feel free to merge the PR even if it doesn't and we can revisit later since `Term type is not valid: ` still highlights the error.

Reply:
That should totally work, but I think we can just skip the inclusion validation when the value is nil. That way we have a little less code to maintain. If you feel strongly about keeping it, I can change it.

Reply:
Skipping validation when nil sounds good to me!